docker-compose.yml

```diff
@@ -0,0 +1,16 @@
+version: "3"
+services:
+  mongo-container:
```

Comment:
This is the important line in the docker-compose file. This names the mongo service as "mongo-container", and docker will create a network and make it available through the DNS name mongo-container. That's why it is the value passed in through the environment variable.